src/lang/en.js

```diff
@@ -81,5 +81,6 @@ AlloyEditor.Strings = {
 	"success": "Success",
 	"table": "Insert Table",
 	"videoPlaybackDisabled": "Video playback is disabled during edit mode",
-	"warning": "Warning"
```

Comment:
If I am reading the Gulp build right, I think we don't update this file by hand. Instead, we update `src/assets/lang/en.json`, and then when you run Gulp, it overwrites `src/lang/en.js`.

Reply:
(Currently working on updating CONTRIBUTING documenting how to do this, and I think I'll add a warning to not update a generated file as well.)

Reply:
Yes you're right ... I updated the wrong file 😭

I'll revert this last commit and update the correct file, but yes updating the CONTRIBUTING document seems like a good idea.

Reply:
Updated in https://github.com/liferay/alloy-editor/pull/1099

Next thing I'll do is add a "do not edit" header at the top and to be honest I may even move them somewhere where it is very obvious that they shouldn't be edited, like in a directory called `src/__generated__` or something...

Reply:
Nice, yes that `src/__generated__` directory sounds like a good idea

Reply:
Ok, went ahead and did that in https://github.com/liferay/alloy-editor/pull/1107